crates/core/src/mock/mock_starknet.rs

```diff
@@ -52,6 +58,7 @@ impl Serialize for AvailableFixtures {
         match self {
             AvailableFixtures::ComputeStarknetAddress => serializer.serialize_str("kakarot_computeStarknetAddress"),
             AvailableFixtures::GetEvmAddress => serializer.serialize_str("kakarot_getEvmAddress"),
+            AvailableFixtures::GetImplementation => serializer.serialize_str("kakarot_getImplementation"),
```

Comment:
not sure to follow, kakarot has no `get_implementation` entrypoint

Reply:
gotcha, will change this should to account instead of kakarot